writing-specifications.rst

```diff
@@ -63,13 +68,11 @@ Example
 
 
 Specification Heading
-+++++++++++++++++++++
+^^^^^^^^^^^^^^^^^^^^^
 
-A specification must begin with a spec heading and a single specification can
-contain only one spec heading.
+A specification must begin with a Spec Heading. A specification contains only one Spec Heading.
```

Comment:
> A specification contains only one Spec Heading.

With this sentence It's not clear that there is a restriction of only one Spec heading allowed per specification.

Reply:
Okay. How about "A specification must contain only one Spec heading."?